components/chassis.py

```diff
@@ -402,12 +401,10 @@ def on_enable(self) -> None:
         self.update_alliance()
         self.update_odometry()
 
-    @magicbot.feedback
-    def get_imu_speed(self) -> float:
-        return math.hypot(self.imu.getVelocityX(), self.imu.getVelocityY())
-
     def get_rotational_velocity(self) -> float:
-        return math.radians(-self.imu.getRate())
+        return math.radians(
+            self.imu.get_angular_velocity_z_world().value
+        )  # TODO Check direction of positive rotation
```

Comment:
nit: put the comment before the line it's about, rather than formatting like this